src/game/frontend/Menu.cpp

```diff
@@ -1,5 +1,6 @@
 #include "Menu.hpp"
 
+#include "game/rdr/Natives.hpp"
```

Comment:
Don't see why you would include this and not use it.

Reply:
I was messing with PAD::DISABLE_CONTROL_ACTION to attempt to stop the game from being controlled while the menu is opened, thats why its there.

Reply:
> I would remove it, as it is an unnecessary import

Could, but im still working on that bit on my end, mainly because when the menu is opened, clicking on options makes the player shoot their weapons, also when scrolling through the menu options if there is a scroll to it, it will scroll through the game options (on the main screen)

So disabling that stuff is what im after

Reply:
Ik hes pretty busy, but maybe @Yimura could help achieve this.  I tried putting the native in for disabling control actions in-game when the menu is opened but it entirely disables the mouse in game, regardless if menu is opened or not. 
ik im doing something wrong so before i pushed the commit i removed the disable control portion, but left the native include.